config.md

```diff
@@ -265,6 +265,41 @@ Annotations are key-value maps.
 }
 ```
 
+## VM
+
+VM is an optional object used by virtual machine-based runtimes.
+
+* **`imagePath`** (string, required) path to file that represents the root filesystem for the virtual machine.
+* **`kernel`** (object, required) specifies details of the kernel to boot the virtual machine with.
+
+Note that `imagePath` refers to a path on the host (outside of the virtual machine). This field is distinct from the **`path`** field in the (#Root-Configuration) section since in the context of a virtual machine-based runtime:
```

Comment:
I may be hyphen-happy, but I'm pretty sure a “machine-based” hyphen should also come with a “virtual-machine” hyphen.  Whether “virtual-machine-based” or “virtual-machine based” is better is beyond me ;).